regexes/bots.yml

```diff
@@ -961,7 +961,7 @@
     name: 'BIZON, OOO'
     url: 'https://bi.zone/'
 
-- regex: 'masscan'
+- regex: '(?:ivre-)?masscan'
```

Comment:
Can't we simply do the check without the additional optional check in the front? It doesn't have any value, or does it?

Reply:
It doesn't. I've added `.*`, otherwise, the built regex won't detect it.